manifests/init.pp

```diff
@@ -60,6 +60,7 @@
   $initial_location = $katello_devel::params::initial_location,
   $admin_password = $katello_devel::params::admin_password,
   $enable_ostree = $katello::params::enable_ostree,
+  $candlepin_event_queue = $katello_devel::params::candlepin_event_queue,
```

Comment:
The reason this works is `$::katello_devel::params` inherits from `$::katello::params`.